Sources/StreamChatUI/ChatMessageList/JumpToUnreadMessagesButton.swift

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// A Button that is used to indicate unread messages in the Message list.
+open class JumpToUnreadMessagesButton: _Button, ThemeProvider {
+    /// The unread count that will be shown on the button as a badge icon.
+    var content: ChannelUnreadCount = .noUnread {
+        didSet {
+            updateContentIfNeeded()
+        }
+    }
+
+    /// The view showing number of unread messages in channel if any.
+    open private(set) lazy var textLabel = UILabel()
+        .withoutAutoresizingMaskConstraints
+        .withBidirectionalLanguagesSupport
+        .withAdjustingFontForContentSizeCategory
+
+    open private(set) lazy var closeButton = UIButton().withoutAutoresizingMaskConstraints
```

Comment:
I wonder if we could use a stack view to layout these 🤔 It would simplify the `setUpLayout()` code and it would also make it easier for customers to change the layout without overriding the `setUpLayout()` completely.

Reply:
I am pretty against having an Stack View that customers manually mutate, as it is something that depends on the internal layout, and does not scale well.

On the other hand, it is such a simple view, that maybe it is not worth it to allow customizability within it.

Reply:
I'm with Pol on this one, there's no need to introduce a stack view here.